presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -189,7 +189,10 @@ These properties require some explanation:
   the host and port of the Presto coordinator. This URI must not end
   in a slash.
 
-You may also wish to set the following properties:
+.. _config_properties_jmx:
```

Comment:
Let's move the JMX related properties to the JMX page. I think having them here predates that page, and it was useful to highlight here since it picked a random port otherwise. Now that it's optional, we can leave it all on the JMX page.

Reply:
Sounds good... do we still need to have a mention about JMX on that page then? I am not sure myself.. maybe not.

Reply:
I’d remove it. We only listed it there since it started automatically.

Reply:
Done.